alphagov/paas-product-pages
# replace email address for CDDO with a generic mailbox

use a generic mailbox `cddo-cloud-programme-service-desk@digital.cabinet-office.gov.uk` rather than a specific person

# how to review
- clone the branch run locally using the instructions in the README
- check the copy for typos and syntax
- ensure the link to the CDDO mailbox works

# who can review
- anyone but @pauldougan
